src/riak_core_util.erl

```diff
@@ -150,6 +155,55 @@ replace_file(FN, Data) ->
             Err
     end.
 
+%% @doc Similar to {@link file:consult/1} but will consult files that
+%% are not valid unicode. Specifially files written with
+%% `io_lib:format("~p,", io_list())' in r16 before `utf8' became the
+%% default encoding.
+-spec consult(file:filename()) -> {ok, [term()]} | {error, Err :: term()}.
+consult(FName) ->
+    case file:consult(FName) of
+        {ok, Terms} ->
+            {ok, Terms};
+        {error, _Reason} ->
+            latin_consult(FName)
```

Comment:
I'm assuming this might be used with files we haven't thought of yet, so an encoding marker ***could*** be present in the file. If it is, `epp:set_encoding` won't override it, so we shouldn't bother trying.

The version check is there to make sure the retry uses the opposite encoding from what `file:consult` used, so we're not just performing the same operation the hard way. If run on R16, there's no reason to retry reading it in `latin1` encoding, because `file:consult` already tried that..